tests/integration/test_cli_commands.py

```diff
@@ -0,0 +1,66 @@
+import locale
+import os
+import subprocess
+import sys
+from pathlib import Path
+from typing import Generator, Tuple
+
+import pytest
+
+os.environ["PYTHONIOENCODING"] = "utf-8"
+
+sys.stdout.reconfigure(encoding="utf-8")  # type: ignore
+sys.stderr.reconfigure(encoding="utf-8")  # type: ignore
+
+ONNX_PATH = Path("tests/integration/model.onnx")
+
+
+@pytest.fixture(scope="session", autouse=True)
+def clear_files() -> Generator[None, None, None]:
```

Comment:
Just a note; it's fine not to add return types to the fixtures, they work by magic and adding the hint doesn't really help ,uch (editors/type checkers have no use for it and the generator type is also not very readable for humans)